agent/conf/agent.properties

```diff
@@ -17,277 +17,438 @@
+# Default value: 00:16:3e:77:e2:a0
+#private.macaddr.start=00:16:3e:77:e2:a0
+
+# Specifies start IP address for private IP range.
+# Default value: 192.168.166.128
+#private.ipaddr.start=192.168.166.128
+
+# Defines Local Bridge Name.
+#private.bridge.name=
+
+# Defines private network name.
+#private.network.name=
+
+# Defines the location for network scripts.
+# The path defined in this property is relative.
+# To locate the script, ACS first tries to concatenate
+# the property path with "/usr/share/cloudstack-agent/lib/".
+# If it fails, it will test each folder of the path,
+# decreasing one by one, until it reaches root.
+# If the script is not found, ACS will repeat the same
+# steps concatenating the property path with "/usr/share/cloudstack-common/".
+# The path defined in this property is relative
+# to the directory "/usr/share/cloudstack-common/".
+# Default value: scripts/vm/network/vnet
+#network.scripts.dir=scripts/vm/network/vnet
+
+# Defines the location for storage scripts.
+# The path defined in this property is relative.
+# To locate the script, ACS first tries to concatenate
+# the property path with "/usr/share/cloudstack-agent/lib/".
+# If it fails, it will test each folder of the path,
+# decreasing one by one, until it reaches root.
+# If the script is not found, ACS will repeat the same
+# steps concatenating the property path with "/usr/share/cloudstack-common/".
+# The path defined in this property is relative
+# to the directory "/usr/share/cloudstack-common/".
+# Default value: scripts/storage/qcow2
+#storage.scripts.dir=scripts/storage/qcow2
+
+# Time (in seconds) to wait for the VM to shutdown gracefully.
+# If the time is exceeded shutdown will be forced.
+# Default value: 120
+#stop.script.timeout=120
+
+# Definition of VMs video model type.
+#vm.video.hardware=
+
+# Definition of VMs video, specifies the amount of RAM in kibibytes (blocks of 1024 bytes).
+# Default value: 0
+#vm.video.ram=0
+
+# System VM ISO path.
+#systemvm.iso.path=
+
+# If set to "true", allows override of the properties: private.macaddr.start, private.ipaddr.start, private.ipaddr.end.
+# Default value: false
+#developer=false
+
+# Can only be used if property developer = true. This property is used to define the link local bridge name and private network name.
+#instance=
 
-# Enable manually setting CPU's topology on KVM's VM.
-# enable.manually.setting.cpu.topology.on.kvm.vm=true
+# Shows the path to the base directory in which NFS servers are going to be mounted.
+# Default value: /mnt
+#mount.path=/mnt
 
-# Manually set the host CPU MHz, in cases where CPU scaling support detected value is wrong
-# host.cpu.manual.speed.mhz=0
+# Port listened by the console proxy.
+# Default value: 443
+#consoleproxy.httpListenPort=443
 
-# Enable/disable IO driver for Qemu (in case it is not set CloudStack can also detect if its supported by qemu)
-# enable.io.uring=true
+# Default value: 5
+#ping.retries=5
```

Comment:
why mention all these default values, when these are encoded below? it should be clear these are defaults (and we could add a single remark at the top for it)